rllib/core/models/catalog.py

```diff
@@ -284,14 +282,6 @@ def get_encoder_config(
                     cnn_use_layernorm=model_config_dict.get(
                         "conv_use_layernorm", False
                     ),
-                    output_dims=[encoder_latent_dim],
```

Comment:
This is actually the main TODO that triggered this PR. It was very confusing for users to figure out what we were doing under the hood at the intersection between the CNN stack and the (pi, vf) heads. Now that we removed the auto-dense layer, things are more explicit and clear.

Reply:
100% with you 🙂